pkg/pillar/types/dpc.go

```diff
@@ -401,6 +402,27 @@ func (config *DevicePortConfig) DoSanitize(log *base.LogObject, args DPCSanitize
 				config.Ports[i].IsL3Port = true
 			}
 		}
+
+		sanitizeNTPServers := func(servers *[]string) []string {
```

Comment:
Why do you need to have the argument be a pointer to the slice/arrary?

Reply:
@eriknordmark to avoid a copy of array as i initially thought. but I read just now that copying is not happening here so i can avoid *